puppet/zulip/manifests/base.pp

```diff
@@ -48,6 +48,7 @@
              'missedmessage_mobile_notifications',
              'outgoing_webhooks',
              'signups',
+             'slash_commands',
```

Comment:
It's needed, but only for use in production.  In development, we don't use the `puppet` configuration at all.